x-pack/plugins/observability/public/pages/rules/index.tsx

```diff
@@ -75,6 +80,10 @@ export function RulesPage() {
     field: 'name',
     direction: 'asc',
   });
+  const [inputText, setInputText] = useState<string | undefined>();
+  const [searchText, setSearchText] = useState<string | undefined>();
+  const [refreshInterval, setRefreshInterval] = useState(60000);
```

Comment:
@mgiota, a minuscule thing here,  the auto-refresh default values show 1 min. But when clicking to update the interval, it shows 60 seconds. Obviously, the end result is the same. I'm just being annoying from a user perspective. 😅